tests/unit/src/kakarot/instructions/test_environmental_information.cairo

```diff
@@ -140,3 +140,39 @@ func test__exec_extcodecopy__should_handle_address_with_no_code{
 
     return ();
 }
+
+@view
+func test__returndatacopy{
+    syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*
+}() {
+    // Given
+    alloc_locals;
+
+    let (bytecode) = alloc();
+    let return_data: felt* = alloc();
+    let return_data_len: felt = 32;
+    // filling at return_data + 1 because first first felt is return_data offset
+    TestHelpers._fill_bytecode_with_values(return_data + 1, 32, 0xFF);
```

Comment:
We can use return_data_len here to avoid duplication

Reply:
Also, we can rename the _fill_bytecode_with_values by a more generic name now, for example array_fill()